sklearn/naive_bayes.py

```diff
@@ -62,6 +64,30 @@ def _check_X(self, X):
         Only used in predict* methods.
         """
 
+    def predict_joint_log_proba(self, X):
+        """Return joint log probability estimates for the test vector X.
```

Comment:
```suggestion
        """Return joint log probability estimates for the data X.
```

Reply:
Here I am simply following the suit. The wording "test vector X" has been used in other `predict_*` methods.

Reply:
I agree the phrasing is not ideal, although I personally have more issues with the word "vector".  

Shall we leave it "test vector X" solely for consistency, so that it is easier to correct all methods' docstrings at once in the future? I had a very quick look at how other methods call `X` in `predict`, and it varies. I'd rather refrain from looking into this at this time.